include/gpio.h

```diff
@@ -189,8 +189,29 @@ extern "C" {
  */
 #define GPIO_DS_DISCONNECT_HIGH (0x3 << GPIO_DS_HIGH_POS)
 
+
+
+#if defined(CONFIG_SOC_FAMILY_NRF5)
```

Comment:
gpio.h is meant to be generic among all SoC/Boards, so you cannot add SoC specific features there.

Reply:
thoe values need to come either from the SDK or should be declared in the SOC related files. See ext/hal/nordic/mdk, for example grep for GPIO_PIN_CNF_SENSE_Low